Chocolatey/public/Remove-ChocolateyPin.ps1

```diff
@@ -35,12 +35,12 @@ function Remove-ChocolateyPin {
         }
 
         if (!(Get-ChocolateyPackage -Name $Name)) {
-            Throw "Chocolatey Package $Name cannot be found."
+            Throw "The Pin for Chocolatey Package $Name cannot be found."
         }
 
         $ChocoArguments = @('pin', 'remove', '-r')
         $ChocoArguments += Get-ChocolateyDefaultArgument @PSBoundParameters
-        Write-Verbose "choco $($ChocoArguments -join ' ')"
+        # Write-Debug "choco $($ChocoArguments -join ' ')"
```

Comment:
Either you remove it or you leave it, but probably no need to comment it...
Not a blocker so I'll pass.